tests/scripts/run-pmi-diffs.py

```diff
@@ -0,0 +1,704 @@
+            log('Copy: %s => %s' % (source_pathname, target_pathname))
+            if not testing:
+                shutil.copy2(source_pathname, target_pathname)
+
+##########################################################################
+# Do baseline build:
+# 1. determine appropriate commit,
+# 2. clone coreclr,
+# 3. do build
+##########################################################################
+
+def baseline_build():
+
+    if not testing:
+        if os.path.isdir(baseCoreClrPath):
+            log('Removing existing tree: %s' % baseCoreClrPath)
+            shutil.rmtree(baseCoreClrPath, onerror=del_rw)
+
+    # Find the baseline commit
+
+    # Clone at that commit
+
+    command = 'git clone -b %s --single-branch %s %s' % (
+        target_branch, Coreclr_url, baseCoreClrPath)
+    log(command)
+    returncode = 0 if testing else os.system(command)
+    if returncode != 0:
+        log('ERROR: git clone failed')
+        return 1
+
+    # Change directory to the baseline root
+
+    cwd = os.getcwd()
+    log('[cd] %s' % baseCoreClrPath)
+    if not testing:
+        os.chdir(baseCoreClrPath)
+
+    # Set up for possible docker usage
+
+    scriptPath = '.'
+    buildOpts = ''
+    dockerCmd = ''
+    if not Is_windows and (arch == 'arm' or arch == 'arm64'):
+        # Linux arm and arm64 builds are cross-compilation builds using Docker.
+        if arch == 'arm':
+            dockerFile = 'microsoft/dotnet-buildtools-prereqs:ubuntu-14.04-cross-e435274-20180426002420'
+            dockerOpts = '-e ROOTFS_DIR=/crossrootfs/arm -e CAC_ROOTFS_DIR=/crossrootfs/x86'
+        else:
+            # arch == 'arm64'
+            dockerFile = 'microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-cross-arm64-a3ae44b-20180315221921'
+            dockerOpts = '-e ROOTFS_DIR=/crossrootfs/arm64'
+
+        dockerCmd = 'docker run -i --rm -v %s:%s -w %s %s %s ' % (baseCoreClrPath, baseCoreClrPath, baseCoreClrPath, dockerOpts, dockerFile)
+        buildOpts = 'cross crosscomponent'
+        scriptPath = baseCoreClrPath
+
+    # Build a checked baseline jit 
+
+    if Is_windows:
+        command = 'set __TestIntermediateDir=int&&build.cmd %s checked skiptests skipbuildpackages' % arch
+    else:
+        command = '%s%s/build.sh %s checked skiptests skipbuildpackages %s' % (dockerCmd, scriptPath, arch, buildOpts)
+    log(command)
+    returncode = 0 if testing else os.system(command)
+    if returncode != 0:
+        log('ERROR: build failed')
+        return 1
+
+    # Build the layout (Core_Root) directory
+
+    # For Windows, you need to first do a restore. It's unfortunately complicated. Run:
+    #   run.cmd build -Project="tests\build.proj" -BuildOS=Windows_NT -BuildType=Checked -BuildArch=x64 -BatchRestorePackages
+
+    if Is_windows:
+        command = 'run.cmd build -Project="tests\\build.proj" -BuildOS=Windows_NT -BuildType=%s -BuildArch=%s -BatchRestorePackages' % (build_type, arch)
```

Comment:
Same comment as in the groovy. This logic should be added to one of our windows build scripts.